### PR TITLE
updated bs.json

### PR DESCRIPTION
**Pull request by Hamy-os**

### CODE REVIEW

**Review by jfrader — APPROVED** [review]
Good work!

**Comment by TasoOneAsia** [comment]
Needs changes to `phone/src/i18n.ts` as well for the rename and two-letter code change.

**Review by TasoOneAsia — CHANGES_REQUESTED** [review]
Needs changes to `phone/src/i18n.ts` as well for the rename and two-letter code change.

**Comment by Hamy-os** [comment]
ok done

**Comment by itschip** [comment]
@Hamy-os I see there are more keys that are still in English. Would you mind changing these?

File: phone/src/i18n.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { initReactI18next } from 'react-i18next';
 import en from './locale/en.json';
 import es from './locale/es.json';
 import fr from './locale/fr.json';
-import bh from './locale/bh.json';
+import bh from './locale/bs.json';
```
Comment (TasoOneAsia) [inline phone/src/i18n.ts:6]: Still incorrect, please change the name of default import and the subsequent reference as well

**Review by TasoOneAsia — CHANGES_REQUESTED** [review]

File: phone/src/i18n.ts (hunk 1 above)
Comment (jfrader) [reply phone/src/i18n.ts:6]: Omg how did I miss it lol

Also needs change in i18n.missingKeys script

**Comment by Hamy-os** [comment]
be back soon

**Comment by LedAndris** [comment]
smh hamy

**Pull request closed without merging**